azure-pipelines.yml

```diff
@@ -17,17 +17,11 @@ jobs:
         NUMPY: '1.21'
         CONDA_ENV: 'azure_ci'
         TEST_START_INDEX: 0
-      py310_np124:
-        PYTHON: '3.10'
-        NUMPY: '1.24'
-        CONDA_ENV: 'azure_ci'
-        TEST_THREADING: 'tbb'
```

Comment:
This remove TBB testing on OSX

Reply:
Should probably keep on doing this as it tests a different loading path. Reinstated in 1bd9dd0.